recipes-extended/onl/onl.inc

```diff
@@ -149,7 +146,7 @@ do_compile() {
       V=1 VERBOSE=1 oe_runmake -C ${MODULES_DIR}/builds modules ARCH=${ARCH} KERNELS=${STAGING_KERNEL_BUILDDIR}
   done
 
-  for vendor in ${ONL_MODULE_VENDORS}; do
```

Comment:
Do we still need ONL_MODULE_VENDORS in conf/machine/include/onl.inc?

Reply:
We don't, let me drop that.

Reply:
Dropped.